src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -72,10 +72,39 @@ describe('<AddRemoveForm />', () => {
 
   it('should show recent items only when recentOnly is true')
 
-  it('should show a message when no matches are found')
+  it('should show a message when no matches are found', async () => {
+    const emptyItemsProps = {
+      ...mockProps.questions,
+      fetchItems: () => Promise.resolve([])
+    }
+
+    wrapper = await shallow(<AddRemoveForm {...emptyItemsProps} />)
+    await wrapper.update()
+
+    let result = wrapper.find({ 'data-test': 'none-available-message' })
+    expect(result.length).toBe(1)
+    expect(result.text()).toBe('No  Matches')
+
+    wrapper.instance().toggleRecentOnly({ preventDefault: jest.fn() })
+    result = wrapper.find({ 'data-test': 'none-available-message' })
+    expect(result.length).toBe(1)
+    expect(result.text()).toBe('No Recent Matches')
+  })
```

Comment:
In this test we toggled the `recentOnly` setting by calling the method directly. We could have simulated a click as well, which we will probably do in another test. If that solution turns out to be cleaner, I may update this.

Notice we had to make our instance off `wrapper` and not `result`. We can only make an instance from the root element. This forced us to make our changes, then run another `find()` on the element. Without grabbing the element again, we were still referencing the old element with the old text, causing a failure.